src/pages/Homepage/Homepage.tsx

```diff
@@ -1,6 +1,24 @@
 import { LetsStartExploring } from './LetsStartExploring'
+import { BookCard } from '../../components/BookCard'
 
 const Homepage = () => {
-  return <LetsStartExploring />
+  return (
+    <div>
+      <LetsStartExploring />
+      <div
+        style={{
```

Comment:
Move this styling to Homepage.module.css to keep all styling consistent
Also for the sake of consistency, convert those 10rem to pixels